src/containers/NetworkCheck/NetworkCheck.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react'
 import { Popup } from 'decentraland-ui/dist/components/Popup/Popup'
 import { T } from '../../modules/translation/utils'
-import { getChainIdByNetwork, getConnectedProviderChainId } from '../../lib/eth'
+import { getChainIdByNetwork } from '../../lib/eth'
 import { Props } from './NetworkCheck.types'
-import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
 import ChainProvider from '../ChainProvider'
 
 export default class ChainCheck extends React.PureComponent<Props> {
```

Comment:
Could we rename this to NetworkCheck? I don't think it would be a breaking change because it is exported by default

Reply:
done ✅